src/main/java/seedu/address/model/participation/Participation.java

```diff
@@ -41,6 +41,23 @@ public static Boolean isValidParticipationList(String test) {
         return test.matches(VALIDATION_REGEX);
     }
 
+    @Override
+    public boolean equals(Object other) {
+        if (other == this) {
+            return true;
+        }
+
+        // instanceof handles nulls
+        if (!(other instanceof Participation)) {
+            return false;
+        }
+
+        Participation otherParticipation = (Participation) other;
```

Comment:
I also made this lol guess I will resolve merge conflict later
Actually just for show cause have method 'isSameParticipation' that is used mainly already. This method does not check for equality between attendancelist which is more useful

Reply:
oh yea i saw it in your pr also